Makefile

```diff
@@ -23,7 +23,22 @@ fmt:
 	goimports -w -l -local github.com/autokitteh/starlark-lsp cmd/ pkg/
 
 install:
-	go install ./cmd/starlark-lsp
+	go install ./cmd/autokitteh-starlark-lsp
 
 builtins:
 	go run ./hack/starlark-builtins.go > pkg/analysis/builtins.py
+
+
+.PHONY: sysroot-pack sysroot-unpack release-dry-run release
+
+PACKAGE_NAME          := github.com/autokitteh/autokitteh-starlark-lsp
+GOLANG_CROSS_VERSION  ?= v1.21  # could use latest, but it's better to specify exact Go version
+
+DOCKER_RUN = docker run --rm -e CGO_ENABLED=1 -v /var/run/docker.sock:/var/run/docker.sock -v `pwd`:/go/src/$(PACKAGE_NAME) -v `pwd`/sysroot:/sysroot -w /go/src/$(PACKAGE_NAME)
+GORELEASER_IMAGE = ghcr.io/goreleaser/goreleaser-cross:${GOLANG_CROSS_VERSION}
+
+release-dry-run:
+	@$(DOCKER_RUN) $(GORELEASER_IMAGE) --clean --skip=validate --skip=publish --debug --verbose
+
+release:
```

Comment:
don't you need a release-mac feature for testing instead of releasing all platforms every time ?

Reply:
please elaborate?

Reply:
not a must
but assume you want to create versions on a dev branch and let people test it
so release only a specific arch might be faster and waste less action minuets

Reply:
- there is nothing platform-specific in LSP code, so I see no need to release for specific platform.
- moreover, there won't be frequent releases, therefore I don't really see a reason why to make releases faster and what will be the use case for this.